util/log/log.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
```

Comment:
I agree with leaving out the year (the old glog format didn't have the year either), but let's zero-pad the month and day so the header is fixed-width.